src/library/dao/playlistdao.cpp

```diff
@@ -227,6 +227,24 @@ void PlaylistDAO::deletePlaylist(const int playlistId) {
     }
 }
 
+void PlaylistDAO::deleteAllEmptyPlaylists(PlaylistDAO::HiddenType type) {
```

Comment:
How about a more versatile function `deleteAllPlaylistsWithFewerItems(PlaylistDAO::HiddenType type, int minNumberOfItems)`
and then invoke it with `minNumberOfItems = 1` for this use case?

Reply:
done